src/python/stup/StupCore/StupCore.py

```diff
@@ -116,7 +117,7 @@ def reset(self):
         if self.state.state_nr == StateMachine.RST:
             return
         self.state_machine.set_state(StateMachine.RST)
-        for i in xrange(Config.RST_SEND_TIMES):
+        for i in range(Config.RST_SEND_TIMES):
```

Comment:
`xrange` has a better performance on py2.

Can we use `six.moves.builtins.range`?